cookbook/catalog_structure/overriding_the_orm_product_value.rst

```diff
@@ -269,4 +269,8 @@ Finally, check that your mapping override is correct by launching the following
 
     php app/console doctrine:mapping:info
 
-Now you are ready to perform a Doctrine schema update and use your own `ProductValue` class.
+Now you are ready to perform a Doctrine schema update and use your own `ProductValue` class:
+
+.. code-block:: bash
+
+    php app/console doctrine:schema:update --dump-sql --force
```

Comment:
I assume the developer is in development mode and is aware of environments and cache issues: if it's not the case, this needs a specific chapter somewhere... but not here. What do you think?